src/main/java/com/android/volley/toolbox/CacheHeader.java

```diff
@@ -14,6 +16,9 @@
     /** Magic number for current version of cache file format. */
     private static final int CACHE_MAGIC = 0x20150306;
 
+    /** Bits required to write 4 longs and 3 ints */
```

Comment:
Same here – add a period

Reply:
fyi, looks like this is still unaddressed